java/device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/transport/mqtt/ProtocolOverridingSSLSocketFactory.java

```diff
@@ -0,0 +1,87 @@
+package com.microsoft.azure.iothub.transport.mqtt;
```

Comment:
Can you please add information related to this module in the requirements document and also if possible please add few tests